pyiron_atomistics/lammps/potential.py

```diff
@@ -78,7 +78,9 @@ def files(self):
             resource_path_lst = s.resource_paths
             for conda_var in ["CONDA_PREFIX", "CONDA_DIR"]:
                 if conda_var in env.keys():  # support iprpy-data package
-                    resource_path_lst += [os.path.join(env[conda_var], "share", "iprpy")]
+                    path_to_add = os.path.join(env[conda_var], "share", "iprpy")
+                    if path_to_add not in resource_path_lst:
+                        resource_path_lst.append(path_to_add)
```

Comment:
cf. my larger concerns in the issue, but here at least we also need to cast the path as posix

Reply:
Aren't all operations on this going to be on the filesystem directly via function in `os.*`?  If so they should be system-dependent paths to work on windows as well.

Reply:
💯 yes, pyiron should still run it because we `os.*` on it later, *but* [we claim](https://github.com/pyiron/pyiron_base/blob/97e1d0a523c45720f897a5dee906e3ed9efe8449/pyiron_base/state/settings.py#L98) that we *store* paths in posix form. I'm by no means married to that claim, I just carried it over during my refactor...but as long as those words are there we should stay true to them.

Reply:
Ah, I'll use that then and work on a "nicer" interface elsewhere.